graphene_sqlalchemy/batching.py

```diff
@@ -2,15 +2,28 @@
 from asyncio import get_event_loop
 from typing import Any, Dict
 
-import aiodataloader
 import sqlalchemy
 from sqlalchemy.orm import Session, strategies
 from sqlalchemy.orm.query import QueryContext
 
 from .utils import is_graphene_version_less_than, is_sqlalchemy_version_less_than
 
 
-class RelationshipLoader(aiodataloader.DataLoader):
```

Comment:
RelationshipLoader is pointing to `aiodataloader.DataLoader`, when it should be using the locally-sourced DataLoader that comes from the `get_data_loader_impl()` function.